app/grandchallenge/workstations/static/workstations/js/session-control.mjs

```diff
@@ -0,0 +1,132 @@
+function openWorkstationSession(element) {
+    return (event) => {
+        const windowIdentifier = element.dataset.workstationWindowIdentifier;
+        const url = element.dataset.createSessionUrl;
+        const query = element.dataset.workstationQuery;
+        const creationURI = `${url}?${query}`;
+        const timeout = element.dataset.timeout || 1000;
+
+        if (event.ctrlKey) {
+            window.open(creationURI);
+            return;
+        }
+
+        if (event.altKey) {
+            copyTextToClipboard(query);
+            return;
+        }
+
+        setSpinner(element);
```

Comment:
Maybe put the next part of this function in a try/catch block and put `removeSpinner()` in the catch block so that it always is removed even in case of an unexpected error in the next part. You can then always throw the error again so that sentry will catch it.